src/obfl-specification.html

```diff
@@ -1823,6 +1823,16 @@ <h4 id="L84911" class="include">volume-template</h4>
             number. It can be used in an expression, e.g. <code>(= $volume
             1)</code>.</dd>
           <dd>Default is 'volume'.</dd>
+        <dt>started-volume-first-content-page-number-variable [optional]</dt>
+          <dd>Definition of the name of the variable holding the first page
+            number of the content of the started volume. It can be used in an
```

Comment:
If the attribute would be put on the `toc-entry-on-resumed` element it would be immediately clear what "started volume" means.

Reply:
Done. I've added a "Variables" section at `toc-entry-on-resumed`. This does not mean that we will have such Variables section everywhere in the future - that depends on how issue #14 will be solved. For now this will do.